linux_os/guide/services/xwindows/disabling_xwindows/xwindows_remove_packages/rule.yml

```
@@ -72,6 +64,8 @@ warnings:
         The installation and use of a Graphical User Interface (GUI) increases your attack vector and decreases your
         overall security posture. Removing the package xorg-x11-server-common package will remove the graphical target
         which might bring your system to an inconsistent state requiring additional configuration to access the system
-        again. If a GUI is an operational requirement, a tailored profile that removes this rule should used before
+        again. </br>
+        The rule 'xwindows_runlevel_target' can be used to configure the system to boot into the multi-user.target.</br>
```

```suggestion
        The rule <tt>xwindows_runlevel_target</tt> can be used to configure the system to boot into the multi-user.target.</br>
```

Apparrently we can't use single quotes here.